src/components/SimpleDateChooser.tsx

```diff
@@ -76,16 +80,36 @@ const SimpleDateChooser: React.FC<Props> = (props) => {
     days: number[],
     year: number,
   }[] = [];
+  // TODO: update logic to be able to generate either/or/and past + future
   let startYear = today.year;
   let startMonth = today.month;
-  if (chooseFromPast) {
+  let endYear = today.year;
+  let endMonth = today.month;
+  // Allow past dates
+  if (!dontAllowPast) {
     startMonth -= Math.max(0, numMonthsToShow - 1);
     while (startMonth <= 0) {
       startMonth += 12;
       startYear -= 1;
     }
   }
-  for (let i = 0; i < numMonthsToShow; i++) {
+  // Allow future dates
+  if (!dontAllowFuture) {
+    endMonth += Math.max(0, numMonthsToShow - 1);
+    while (endMonth > 12) {
+      endMonth -= 12;
+      endYear += 1;
+    }
+  }
+  // Calculate total number of months to show
+  let totalMonthsToShow = numMonthsToShow;
+  if (!dontAllowPast && !dontAllowFuture) {
```

Comment:
Is it really as simple as `*2`?

Let's say `numMonthsToShow` is `3` and it's currently May 15th.

If we're showing _both_ past and future:
We'd want to show dates ~3mo earlier (March through May).
We'd want to show dates ~3mo later (May through July).

So the total number of months would be:
March, April, May, June, July = 5 months
The start month would be 3, start year 2025, numMonthsToShow would be 5.

Are we on the same page? I'm not sure if my logic is correct